### PR TITLE
[gitpod-cli] Fix parsing of env variables

### PR DESCRIPTION
**Pull request by aledbf**
fixes https://github.com/gitpod-io/gitpod/issues/4493

### CODE REVIEW

**Comment by aledbf** [comment]
Test:

Screenshot from 2021-06-12 18-29-16

**Review by JanKoehnlein — APPROVED** [review]
LGTM

**Comment by shaal** [comment]
@aledbf Thank you for fixing this so quickly!
Do you know when will it this fix be deployed to gitpod.io ?

**Comment by aledbf** [comment]
@csweichel @meysholdt 👆

File: components/gitpod-cli/cmd/env_test.go [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+			kv, err := parseArgs(test.Input, "")
+			if err != nil {
+				if test.Error != err.Error() {
+					t.Errorf("expected error '%v' but '%v' was returned", test.Error, err)
+				}
+
+				return
+			}
+
+			if !reflect.DeepEqual(kv, test.Expectation) {
```
Comment (csweichel) [inline components/gitpod-cli/cmd/env_test.go:53]: Nit: most tests use go-cmp for this work.

**Comment by csweichel** [comment]
Up to @meysholdt ... Considering current deployment efforts I'd reckon we'd want to batch this change with others. But that's Mo's call.

File: components/gitpod-cli/cmd/env_test.go (hunk 1 above)
Comment (aledbf) [reply components/gitpod-cli/cmd/env_test.go:53]: Please review https://github.com/gitpod-io/gitpod/pull/4505

**Pull request merged**